.ci/Jenkinsfile

```diff
@@ -250,12 +250,13 @@ def checkTestSuite(Map parallelTasks = [:], Map item = [:]) {
   def suite = item.suite
   def platforms = item.platforms
   item.scenarios.each { scenario ->
+    def pullRequestFilter = scenario.pullRequestFilter
```

Comment:
If the key is not present, what value will be passed here? Should we add defensive code here too?